### PR TITLE
Create linter rule for id/name checking

### PR DESCRIPTION
**Pull request by chrisse27**

### CODE REVIEW

File: lib/rules/tag-req-id-or-name.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    process: proc.arrayOfStr
+};
+
+module.exports.lint = function (element, opts) {
+    var format = opts[this.name];
+    if (format.indexOf(element.name) < 0) {
+        return [];
+    }
+
+    if (knife.hasNonEmptyAttr(element, 'id') || knife.hasNonEmptyAttr(element, 'name')) {
```
Comment (acramatte) [inline lib/rules/tag-req-id-or-name.js:22]: @chrisse27 The branch where the element has an id is not covered by tests

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
-  "name": "htmllint",
-  "version": "0.6.0",
+  "name": "@comerge/htmllint",
```
Comment (acramatte) [inline package.json:2]: @chrisse27 Does this go to public registry or ours ?

File: package.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
...
+  "version": "0.6.1",
```
Comment (acramatte) [inline package.json]: @chrisse27 maybe bump to 0.7.0 instead ?

File: lib/rules/tag-req-id-or-name.js [hunk 4]
```diff
@@ -0,0 +1,27 @@
+var knife = require('../knife'),
+    Issue = require('../issue'),
+    proc = require('../processOption');
+
+module.exports = {
+    name: 'tag-req-name',
+    on: ['dom'],
+    filter: ['tag', 'style', 'script'],
```
Comment (acramatte) [inline lib/rules/tag-req-id-or-name.js]: are 'style' and 'script' needed ?

Comment (chrisse27) [reply lib/rules/tag-req-id-or-name.js]: Copy pasted from another tag related rule. Probably not.

File: lib/rules/tag-req-id-or-name.js (hunk 1 above)
Comment (chrisse27) [reply lib/rules/tag-req-id-or-name.js:22]: Will fix that.

File: package.json (hunk 2 above)
Comment (chrisse27) [reply package.json:2]: ours

File: package.json (hunk 3 above)
Comment (chrisse27) [reply package.json]: Yeah, wasn't sure. 0.7.0 it is.

**Comment by chrisse27** [comment]
@acramatte Integrated your requests. Please consider again.

**Review by acramatte — APPROVED** [review]
LGTM

**Pull request merged**